app/models/clinical_resource_hub.rb

```diff
@@ -39,4 +39,27 @@ def self.cached_clinical_resource_hubs
       ClinicalResourceHub.all
     end
   end
+
+  def practices_created_by_crh_count
+    PracticeOriginFacility.where(clinical_resource_hub_id: id).count
+  end
+
+  def practices_adopted_by_crh_count
+    DiffusionHistory.where(clinical_resource_hub_id: id).count
+  end
+
+  def get_crh_adopted_practices( crh_id, options = { :is_user_guest => true })
```

Comment:
let's avoid using the rockets `=>` when we can instead we can do `is_user_guest: true` same comment for other parts we are using `=>`

Reply:
Got it.